bin/sandbox

```diff
@@ -53,10 +53,10 @@ fi
 cd ./sandbox
 cat <<RUBY >> Gemfile
 
+gem 'awesome_nested_set', github: 'peterberkenbosch/awesome_nested_set', branch: 'rails-7-support'
```

Comment:
Now that https://github.com/collectiveidea/awesome_nested_set/pull/460 has been merged, why not using:

```
gem 'awesome_nested_set', github: 'collectiveidea/awesome_nested_set'
```

as you did in the Gemfile?